app/views/events/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= stylesheet_link_tag 'application' %>
+<h1>Events Page</h1>
+<p>Events page stuff goes here</p>
+
+<%# Display all events %>
+<table>
+
+  <thead>
+    <tr>
+      <th>Event Name</th>
+      <th>Description</th>
+      <th>Date</th>
+      <th>Points</th>
+      <th></th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @events.each do |event| %>
+      <tr>
+        <td><%= event.event_name %></td>
+        <td><%= event.description %></td>
+        <td><%= event.date %></td>
+        <td><%= event.points %></td>
```

Comment:
Should we be showing points here?

Reply:
We can show the members how many points an event is worth.